scripts/demo_example.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+from __future__ import print_function
```

Comment:
I'm with @dcolligan here --- the code checked into our repo here should conform to our style guidelines and be consistent with the rest of our code base. I agree on both the imports and the `__main__` bit.